test/suites/dev/moonbase/test-xcm-v3/test-mock-hrmp-transact-ethereum-8.ts

```diff
@@ -92,11 +92,9 @@ describeSuite({
           },
         ];
 
-        let expectedTransferredAmountPlusFees = 0n;
-
-        const targetXcmWeight = 5_000_000_000n + 25_000_000n;
-        const targetXcmFee = targetXcmWeight * 50_000n;
+        const targetXcmFee = 25_000_000n;
```

Comment:
Why this huge difference?

Reply:
We were just passing a huge amount, must of the fees in xcm tests don't make any since. Probably because the tests were added a long time ago and incorrectly updated many times. We should revisit all these xcm tests and improve them in a future maintenance task.